coordinator/gscoordinator/kubernetes_launcher.py

```diff
@@ -209,10 +219,15 @@ def __init__(self, config: Config):
         # frontend port
         self._interactive_port = 8233
         # 8000 ~ 9000 is exposed
-        self._learning_start_port = 8000
+        self._graphlearn_start_port = 8000
+        # 9001 ~ 10001 is exposed
+        self._graphlearn_torch_start_port = 9001
 
         self._graphlearn_services = {}
-        self._learning_instance_processes = {}
+        self._graphlearn_instance_processes = {}
+
+        self._graphlearn_torch_services = {}
+        self._graphlearn_torch_instance_processes = {}
 
```

Comment:
Can graphlearn and graphlearn_torch share the same set of port/processes/services to make these part simpler?

Reply:
We will deprecate graphlearn later, so the separated implementation for now is fine :)